targets/TARGET_STM/TARGET_STM32WL/i2c_device.h

```diff
@@ -34,31 +42,14 @@ extern "C" {
 #define I2CAPI_I2C3_CLKSRC RCC_I2C3CLKSOURCE_SYSCLK
 
 /*  Provide the suitable timing depending on requested frequency */
```

Comment:
is the comment still correct if the function was removed?

Reply:
@0xc0170 This can be removed. I have somehow missed.